libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -191,6 +191,8 @@
 #define HAL_OS_SOCKETS 1
 #define HAL_STORAGE_SIZE            16384
 #define HAL_STORAGE_SIZE_AVAILABLE  HAL_STORAGE_SIZE
+#define HAL_OPTFLOW_PX4FLOW_I2C_ADDRESS 0x42    // 7-bit address. 8-bit address is 0x84, range 0x42 - 0x49
```

Comment:
It would be better do define the address if it's not already defined by board subtypes